rules/windows/process_creation/win_sus_auditpol_usage.yaml

```diff
@@ -0,0 +1,27 @@
+title: Suspicious Auditpol Usage 
+id: 0a13e132-651d-11eb-ae93-0242ac130002
```

Comment:
You can delete this file as this rule is already in the public repo

Reply:
https://github.com/SigmaHQ/sigma/blob/1f8e37351e7c5d89ce7808391edaef34bd8db6c0/rules/windows/process_creation/proc_creation_win_sus_auditpol_usage.yml